src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariable.java

```diff
@@ -0,0 +1,204 @@
+/*
+UnusedLocalVariable
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unusedlocalvariable;
+
+import java.util.ArrayList;
+import java.util.function.Predicate;
+
+public class InputUnusedLocalVariable {
```

Comment:
Please add test cases for try with resources (passing and violation). Make sure to also have cases with multiple resources.